filters/ThirdParty/filter_227_List-KR/template.txt

```diff
@@ -1,2 +1,4 @@
 !
-@include "https://raw.githubusercontent.com/List-KR/List-KR/master/filter-AdGuard.txt" /exclude="../../exclusions.txt"
\ No newline at end of file
+@include "https://raw.githubusercontent.com/List-KR/List-KR/master/filter-AdGuard.txt" /exclude="../../exclusions.txt"
```

Comment:
Why is it required? I see it exists in https://github.com/List-KR/List-KR/commit/f9b8cd180b3d72b0ea4d5a16cfd3ba6579d98339#diff-4cf7963c9d3cb7d9b7d2e8f7b0687f8cd5021e85d1900dd69741e8f3bc9efc29

Reply:
`trustLevel` of List-KR is `high` disallowing any javascript rule.

Reply:
Better discuss in our chat.